addons/test_mail/tests/test_mail_message.py

```diff
@@ -70,16 +69,14 @@ def test_empty_message(self):
         note_subtype.sudo().write({'description': 'Very important discussions'})
         record._message_update_content(message, '', [])
         self.assertFalse(message.attachment_ids)
-        self.assertEqual(message.notified_partner_ids, self.partner_admin)
         self.assertEqual(message.starred_partner_ids, self.partner_admin)
         self.assertFalse(message.sudo()._filter_empty(), 'Subtype with description')
 
-        # Completely void now
+        # Completely emptied now
         note_subtype.sudo().write({'description': ''})
         self.assertEqual(message.sudo()._filter_empty(), message)
         record._message_update_content(message, '', [])
-        self.assertFalse(message.notified_partner_ids)
-        self.assertFalse(message.starred_partner_ids)
+        self.assertEqual(message.starred_partner_ids, self.partner_admin)  # starred messages stay (albeit content is removed)
```

Comment:
Why keep checking starred but not notifications?
We might as well keep asserting both.

Reply:
I change to assert `notified_partner_ids` too. I removed it because I was afraid this bloats tests when not really worth keeping but ok otherwise.